pylint/checkers/base.py

```diff
@@ -1098,11 +1104,29 @@ def visit_exec(self, node):
         """just print a warning on exec statements"""
         self.add_message('exec-used', node=node)
 
-    @utils.check_messages('eval-used', 'exec-used', 'bad-reversed-sequence')
+    def _check_misplaced_format_function(self, call_node):
+        if not isinstance(call_node.func, astroid.Attribute):
+            return
+        if call_node.func.attrname != 'format':
+            return
+
+        expr = utils.safe_infer(call_node.func.expr)
+        if isinstance(expr, type(astroid.Uninferable)):
```

Comment:
You can write this line as `expr is astroid.Uninferable` since all references to Uninferable are the same object